routers/web/org/teams.go

```diff
@@ -231,22 +246,28 @@ func NewTeamPost(ctx *context.Context) {
 	ctx.Data["PageIsOrgTeamsNew"] = true
 	ctx.Data["Units"] = unit_model.Units
 	var includesAllRepositories = form.RepoAccess == "all"
+	var unitPerms = getUnitPerms(ctx.Req.Form)
+	var p = perm.ParseAccessMode(form.Permission)
+	if p < perm.AccessModeAdmin {
+		p = unit_model.MinUnitAccessMode(unitPerms)
```

Comment:
It's better to have a comment here about "why using min access mode, but not max"

Reply:
Added some comments.